Packs/OPNSense/Integrations/OPNSense/OPNSense.yml

```
@@ -0,0 +1,975 @@
+commonfields:
+  id: OPNSense
+  version: -1
+name: OPNSense
+display: OPNSense
+category: Network Security
+description: >-
+  Manage OPNsense Firewall. 
+
+  For more information see OPNsense documentation.
+
+  OPNsense is an open source, easy-to-use and easy-to-build HardenedBSD based firewall and routing platform.
+configuration:
+- name: url
+  display: Server URL (e.g. https://opnsense.mydomain.ltd)
+  required: true
+  defaultvalue: https://opnsense.mydomain.ltd
+  type: 0
+  additionalinfo:
+- name: apikey
+  display: API Key
+  required: true
+  defaultvalue:
+  type: 4
+  additionalinfo:
+- name: apisecret
+  display: API Secret
+  required: true
+  defaultvalue:
+  type: 4
+  additionalinfo:
+- name: insecure
+  display: Trust any certificate (not secure)
+  required: false
+  defaultvalue: ""
+  type: 8
+  additionalinfo:
```

```suggestion
  type: 8
```
If there is no `defaultvalue` and no `additionalinfo`, you can remove those fields.